### PR TITLE
Part 2: Clean up actions

### PR DESCRIPTION
**Pull request by lancetipton**
> This is part 2 of keg-herkin updates
> All these PRs share the same tests
> But the code has been broken up into smaller PRs for easier review

## Context

* Keg-Herkin actions needs some cleanup work

## Goal

* Clean up the keg-herkin tap actions 
* Add actions for definitions types
* Clean up actions for sockr

## Updates

* Updated definitions actions
  * Added active definition key to the store
  * Will eventually allows editing a definition and feature file at the same time 
* Normalized sockr actions
  * Made them more consistent 
  * Now all actions use the same model for defining the cmd output
* Cleaned up results screen action
  * Now auto-sets the activeFile of the previous tab
  * Used when auto-switching to run tests from a previous tab  
* Removed a number of files that are no longer used

## Testing

* Do the tests in [this PR](https://github.com/simpleviewinc/keg-herkin/pull/32)

### CODE REVIEW

File: repos/tap/actions/definitions/local/upsertDefinitions.js [hunk 1]
```diff
@@ -13,23 +13,25 @@ const { CATEGORIES } = Values
...
 
   // Sort the definitions by type ( given | then | etc... )
   // This makes it easier to match to features when editing
-  dispatch({
-    type: ActionTypes.UPSERT_ITEMS,
-    payload: {
-      category: CATEGORIES.DEFINITION_TYPES,
-      items: definitionTypes || definitionsByType(definitions),
-    },
-  })
```
Comment (mpcarolin) [inline repos/tap/actions/definitions/local/upsertDefinitions.js:33]: as a reminder, you could also use the helper actions we have in `keg-core` to simplify these basic actions:

```js
import { upsertItems } from 'SVActions'
export const upsertDefinitions = (definitions, types) => {
  upsertItems(CATEGORIES.DEFINITIONS, definitions)
  upsertItems(CATEGORIES.DEFINITION_TYPES, definitionTypes || definitionsByType(definitions))
}
```

and the same thing for `setDefinitions`

**Review by mpcarolin — APPROVED** [review]
Looks good, just left one suggestion for simpler code

File: repos/tap/actions/definitions/local/upsertDefinitions.js (hunk 1 above)
Comment (lancetipton) [reply repos/tap/actions/definitions/local/upsertDefinitions.js:33]: Forgot all about these actually, I'll update to use these

Comment (lancetipton) [reply repos/tap/actions/definitions/local/upsertDefinitions.js:33]: updated

**Pull request merged**